src/mozilla-cookie-helper.js

```diff
@@ -78,7 +78,8 @@ var CookieHelper = {
             encodeURIComponent(sKey) +
             '=; expires=Thu, 01 Jan 1970 00:00:00 GMT' +
             (sDomain ? '; domain=' + sDomain : '') +
-            (sPath ? '; path=' + sPath : '');
+            (sPath ? '; path=' + sPath : '') +
+            '; samesite=lax';
```

Comment:
**issue (non-blocking):** the original issue says we will allow the user to optionally set the samesite value in this method and default to `samesite=lax`, but in this implementation, we are not giving them the option to set a different value

**suggestion**: add a line in the docs to explain why we hardcode `samesite=lax`. Add a test to confirm this is always in use on `removeCookie`